app/helpers/config/site_helper.rb

```diff
@@ -39,6 +39,10 @@ def site_business_resource_center_url
     Settings.site.business_resource_center_url
   end
 
+  def link_to_site_business_resource_center
+    link_to "Business Resource Center", site_business_resource_center_url
```

Comment:
did not created a settings key value fetch because it is more simpler to use string rather than defining a method and access key.